roles/wildfly_systemd/tasks/yml_config_version_prereq.yml

```diff
@@ -16,6 +16,7 @@
 - name: "Execute only if eap_enable is true."
```

Comment:
please update task name to execute only on eap 7.x

Reply:
Fixed it now. Please review